config/settings.yml

```diff
@@ -36,4 +36,9 @@ acceptable_bcp47_codes:
   - fa-Arab
   - fa-Latn
   - fr
+  - ms
   - tr
+  - tr-Arab
```

Comment:
I thought we /just/ removed `tr-Arab` and `tr-Latn`?

Reply:
@jacobthill ?

Reply:
We did just remove those codes in favor of 'tr' because nobody writes Turkish in Arabic script (Ottoman Turkish) anymore so I assumed, incorrectly, that we wouldn't have metadata in Ottoman Turkish. We do, however, because the title and creator field sometimes come in that form. My apologies, but we will need them after all.

Reply:
Thanks @jacobthill; are you saying `tr` and `tr-Latn` are effectively equivalent, then?

Reply:
Yes, I was assuming only Turkish in Latin script so everything currently using `tr` is actually `tr-Latn`

Reply:
And we'll remediate the things that are currently `tr` to actually be `tr-Latn` and then remove `tr`, or is there some compelling reason to keep both despite the potential for confusion?

Reply:
No need to keep `tr`, these can all be changed to `tr-Latn`